eo-parser/src/main/java/org/eolang/parser/Syntax.java

```diff
@@ -50,6 +56,11 @@
  */
 public final class Syntax {
 
+    /**
+     * Line break.
+     */
+    private static final String NEW_LINE = "\n";
```

Comment:
@andreoss what's the point of making it a const? You use it only once. The second usage can be removed. Check this out https://www.yegor256.com/2015/07/06/public-static-literals.html

Reply:
@yegor256 Removed